src/pages/ServiceDetails/ServiceInfo/types/SubsetValidator.ts

```diff
@@ -8,26 +8,13 @@ export default class SubsetValidator {
   }
 
   public isValid() {
-    return this.hasValidName() && this.hasValidLabels();
+    return this.hasValidName();
```

Comment:
Does it make sense to keep the whole Subset validation and the DestinationRuleValidation into the UI?
I guess that it was put in place before Galley. Now, the case that created the motivation to put in place this UI validation is not valid anymore.

https://issues.redhat.com/browse/KIALI-2015

Besides that, the way we show DR and VS is no longer verbose (showing subsets, trafficpolicy, etc https://github.com/kiali/kiali-ui/pull/1901) and therefore we don't have problems accessing to the fields validated here.

Lastly, in case this validator or the DRValidator generates a message, the status column only shows the aggregated validation information (N objects validated, 1 warning found) and we would be at the same place: showing a warning in the list but not in the object details page.

So I guess that I would remove both SubsetValidator, DestinationRuleValidator and the  `addFormatValidation` from the ServiceInfo.

Reply:
> Does it make sense to keep the whole Subset validation and the DestinationRuleValidation into the UI

Yes, +1, I was unsure, but thanks for confirming, I know this was something added before Galley was catching these errors.

I'll remove it.